projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -0,0 +1,104 @@
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+import { __experimentalInspectorPopoverHeader as InspectorPopoverHeader } from '@wordpress/block-editor';
+import { Button, PanelRow, Dropdown, VisuallyHidden } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+import { PostVisibilityCheck } from '@wordpress/editor';
+import { __, sprintf } from '@wordpress/i18n';
+import { META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS } from './constants';
+
+export const accessOptions = {
+	everybody: {
+		label: __( 'Everybody', 'jetpack' ),
+		info: __( 'Visible to everyone.', 'jetpack' ),
+	},
+	subscribers: {
+		label: __( 'All subscribers', 'jetpack' ),
+		info: __( 'Visible to everyone to subscribes to your site.', 'jetpack' ),
```

Comment:
small typo:

```suggestion
		info: __( 'Visible to everyone that subscribes to your site.', 'jetpack' ),
```

Reply:
Thanks, resolved in [2701a7c](https://github.com/Automattic/jetpack/pull/26417/commits/2701a7cc3a0d2c902ea4e7656cd8d5136ab037e3)